hack/openapi.sh

```diff
@@ -19,5 +19,4 @@ source ./hack/go-path.sh
 # get the openapi binary
 command -v openapi-gen >/dev/null || go build -o "${GOPATH}"/bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
 echo "Generating openapi files"
-openapi-gen --logtostderr=true -o "" -i ./api/v1beta1 -O zz_generated.openapi -p ./api/v1beta1 -h ./hack/boilerplate.go.txt -r "-"
-openapi-gen --logtostderr=true -o "" -i ./api/kafka/v1beta1 -O zz_generated.openapi -p ./api/kafka/v1beta1 -h ./hack/boilerplate.go.txt -r "-"
+openapi-gen --logtostderr=true -v 1 -o "" -i github.com/kiegroup/kogito-cloud-operator/api/v1beta1 -O zz_generated.openapi -p ./api/v1beta1 -h ./hack/boilerplate.go.txt -r "-"
```

Comment:
Thanks for this :)
Question: Do we not want to generate openapi files for kafka anymore?

Reply:
There's nothing in there, and we do not offer the CRD to the cluster. Strimzi does it. We only use the types since they are a java operator.

Reply:
Okay🙂